src/templates/video_player.html

```diff
@@ -0,0 +1,43 @@
+{% extends "layout.html" %}
+
+{% block body %}
+<section class="content">
+  <div class="container">
+    <a href="/">
+      <img class="logo" src="/static/images/logo.svg" alt="Emmental Logo">
+    </a>
+    <div class="user-buttons">
+      <a href="/">
+        <button class="logout-button" type="button">Manage</button>
+      </a>
+      <a href="/logout">
+        <button class="logout-button" type="button">Logout</button>
+      </a>
+    </div>
+    <!-- needs a button for main page-->
```

Comment:
Emmental logo goes to the main page

Reply:
Yes. I need to remove that comment. I ended up making a button that goes back to the 'video index' page. Needed a way to back to previous page.